test/test_unary_ufuncs.py

```diff
@@ -19,9 +234,34 @@
 # See NumPy's universal function documentation
 # (https://numpy.org/doc/1.18/reference/ufuncs.html) for more details
 # about the concept of ufuncs.
+
```

Comment:
TODO: add a test for out=, out= on different device, different dtype, mismatched size, correct size, 0 size

Reply:
`out=` for broadcasted `out`